phsyn-t3/cod-u8776t_phsyn-t3.xml

```diff
@@ -57,881 +57,1053 @@
+
+          <p xml:id="phsyn-t3-d1e362"> Item.
+            <lb ed="#T"/> Queritur de hoc sophismate POSSIBILE EST SORTEM VIDERE <add place="above-line">TANTUM</add> OMNEM HOMINEM
+            <lb ed="#T"/> NON VIDENTEM SE. Probatio. Possibile est Sortem videre <add place="above-line">tantum</add> omnem hominem
+            <lb ed="#T"/> cecum. Sed omnis homo cecus est homo non videns se. Ergo possibile est
+            <lb ed="#T"/> Sortem videre <add place="above-line">tantum</add> omnem hominem non videntem. <pc type="#pilcrow"/>Contra. Si possibile est,
+            <lb ed="#T"/> ponatur: Sortes videt <add place="above-line">tantum</add> omnem hominem videntem se. Tunc aut ´
+            <lb ed="#T"/> Sortes videt se aut non videt se. Si non videt se, ergo est
+            <lb ed="#T"/> homo non videns se. Et videt quemquam talem. Ergo videt se.
+            <lb ed="#T"/> Si videt se, ergo est homo videns se. Sed nullum talem
+            vide<lb ed="#T"/>bat. Ergo non videt, <add place="above-line">se</add> quoniam tantum videbat non videntem se.</p>
+
+          <p xml:id="phsyn-t3-d1e366"><pc type="#pilcrow"/>Solutio. Prima est impossibilis, hec scilicet ‘ possibile est Sortem videre tantum omnem
+            <lb ed="#T"/> hominem non videntem Se’. Et ista similiter ‘Sortes videt tantum omnem
+            <lb ed="#T"/> hominem non videntem se <!--there is a marginalia here: "et omnis propositio implicans contradictione est impossibilis propositio"!-->, quia utraque habet in se
+            contradictionem, Ut iam
+            <lb ed="#T"/> patebit. Probatio autem peccat secundum
+            <seg type="correction">consequens
+            <subst>
+              <del></del>
+              <add>con</add>
+            </subst>
+          </seg> ab inferiori ad
+            <lb ed="#T"/> superius cum distributione et etiam cum dictione exclusiva. Ut ‘omnis homo; ergo
+            <lb ed="#T"/> omne animal’ et ‘tantum omnis homo; ergo tantum omne animal’, quia homo non
+            vi<lb ed="#T"/>dens se in plus est quam homo cecus. Unde sequitur non conversim. Ut ‘Si
+            <lb ed="#T"/> est homo cecus, est homo non videns Se’ et non econverso, quia homo
+            dormi<lb ed="#T"/>ens est homo non videns se9 et etiam vigilans <add place="above-line">habens</add> oculos clausos
+            <lb ed="#T"/> est homo non videns se.</p>
+
+          <p xml:id="phsyn-t3-d1e369">Quod ista ‘Sortes videt <add place="above-line">tantum</add> omnem hominem non
+          <lb ed="#T"/>videntem se’ habeat contradictionem in se patet, quia equipollet istis
+            dua<lb ed="#T"/>bus, scilicet ‘Sortes videt omnem hominem non videntem se’ et ‘non
+            <lb ed="#T"/> videt aliquem hominem <seg type="erasure">tantum</seg> <!--according to Rijk !--> videntem se’, et iste due sunt expositive illius.
+            <lb ed="#T"/> Sed <seg type="correction">ex<subst>
+              <del></del>
+              <add>ex</add>
+            </subst>
```

Comment:
If a whole word is being corrected, you do not need the `<seg type="correction">`. 

Here it should just be: 

`<subst><del rend="erasure"></del><add>ex</add><subst>`